test/kata6.joinNames.test.js

```diff
@@ -1,5 +1,35 @@
 const { joinNames } = require("../src");
 
 describe("joinNames", () => {
-  test("returns string of names, seperated by commas and an ampersand", () => {});
+  test("returns string of names, seperated by commas and an ampersand", () => {
+    expect(
+      joinNames([
+        {
+          name: "Bart",
+        },
+        {
+          name: "Lisa",
+        },
+        {
+          name: "Maggie",
+        },
+      ])
+    ).toEqual("Bart, Lisa & Maggie");
+    expect(
+      joinNames([
+        {
+          name: "Cloud",
+        },
+        {
+          name: "Tifa",
+        },
+        {
+          name: "Zack",
+        },
+        {
+          name: "Aeris",
+        },
+      ])
+    ).toEqual("Cloud, Tifa, Zack & Aeris");
```

Comment:
You should add more tests for edge cases, i.e. when the array of namesObj has length 1 or 2.

When the array length is 1 you currently get the wrong output.

When the array length is 2 you get the expected output, but you should still have test for this edge case just to confirm this and to ensure that the functionality doesn't break if you were to refactor it.